Configurations/WebKitProjectPaths.xcconfig

```diff
@@ -70,6 +70,11 @@ UMBRELLA_FRAMEWORKS_DIR = $(UMBRELLA_FRAMEWORKS_DIR_OVERRIDE_$(WK_USE_OVERRIDE_F
 UMBRELLA_FRAMEWORKS_DIR_OVERRIDE_NO = $(INSTALL_PATH_PREFIX)$(NORMAL_UMBRELLA_FRAMEWORKS_DIR);
 UMBRELLA_FRAMEWORKS_DIR_OVERRIDE_YES = $(WK_OVERRIDE_FRAMEWORKS_DIR);
 
+// Modified build directory for creating separate Safari and WebKit archives from the same build.
+CONFIGURATION_BUILD_DIR = $(CONFIGURATION_BUILD_DIR_$(WK_PROJECT_SPECIFIC_BUILD_DIRS));
+CONFIGURATION_BUILD_DIR_ = $(BUILD_DIR)/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME);
+CONFIGURATION_BUILD_DIR_YES = $(BUILD_DIR)/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)/webkit;
```

Comment:
`$(BUILD_DIR)/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)` is the default value. I tried using `$(inherited)` earlier, had trouble getting it working—it seemed like it was being re-evaluated multiple times—and I didn't investigate any further.